README.md

```diff
@@ -1,2 +1,144 @@
 # pointcloudToMesh
+
 Go Viam module for converting a pointcloud into a spatialmath.mesh
+
+notes on what I want this module to be capable of
+
+1. we take in a pointcloud
+2. we do outlier removal on the pointcloud
+3. (for not optional) we achieve uniform density of points
+4. we return that as a pointcloud
+5. we turn the pointcloud into a mesh
+6. we return the mesh
+
+PUT THIS IN THE RDK YOU ARE RUNNING
```

Comment:
Rather than this, might be easier to just say "make sure go.mod points at an RDK version which includes commit 4e01070736b34f7f3eca9106dc3fa22dccf6ff5f"